ncm-ssh/src/main/pan/components/ssh/schema.pan

```diff
@@ -117,6 +117,8 @@ type ssh_daemon_type = {
     "options" ? ssh_daemon_options_type
     "comment_options" ? ssh_daemon_options_type
     "sshd_path" ? string
+    # if true and sshd doesn't exist, will skip config validation
```

Comment:
or put the comment in `@{}`, so it will at least be picked up by the annotations

@ned21 but what is the inverse/opposite of always? never or optional